tests/hazmat/primitives/test_rsa.py

```diff
@@ -67,6 +67,15 @@ class DummyMGF(object):
     _salt_length = 0
 
 
+@pytest.fixture
+def disable_rsa_checks(backend):
+    # Use this fixture to skip RSA key checks in tests that need the
+    # performance.
+    backend._rsa_skip_check_key = True
+    yield
+    backend._rsa_skip_check_key = False
```

Comment:
```suggestion
    try:
        yield
    finally:
        backend._rsa_skip_check_key = False
```

Reply:
pytest does magic here and exceptions inside the yield are caught and the statements after are executed, so there's no need for a try/finally here.